dlp/snippets/jobs/jobs_test.go

```diff
@@ -119,7 +119,7 @@ func riskNumerical(projectID, dataProject, pubSubTopic, pubSubSub, datasetID, ta
 	// Create the risk job.
 	j, err := client.CreateDlpJob(ctx, req)
 	if err != nil {
-		return fmt.Errorf("CreateDlpJob: %w", err)
+		return fmt.Errorf("CreateDlpJob: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.